corehq/apps/domain/management/commands/calculate_physical_size.py

```diff
@@ -17,7 +17,7 @@ class Command(BaseCommand):
 
     def add_arguments(self, parser):
         parser.add_argument(
-            'domain',
+            'domains',
```

Comment:
Should we add a help block here on how we are expecting the domain list to be?
I see that we are splitting by spaces
```
domains_list = domains.split()
```
I think it would be a good idea to mention what the argument expects.

Reply:
That's a good point Amit. I will make the changes both you and Matt suggested and update the pr.